packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -120,6 +120,7 @@ function ScrollView(props: ScrollViewProps, ref: RefObject<HTMLDivElement>) {
     return () => {
       clearTimeout(state.scrollTimeout);
     };
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
shouldn't this be `};` since we aren't asking the useEffect to listen for any changes?

Reply:
nah, only want this on mount I believe, not every render